netket/vqs/base.py

```diff
@@ -159,7 +160,7 @@ def expect(self, Ô: AbstractOperator) -> Stats:
         Returns:
             An estimation of the quantum expectation value <O>.
         """
-        raise NotImplementedError
+        return expect(self, Ô)
 
     def grad(
         self, Ô, *, is_hermitian: Optional[bool] = None, mutable: Optional[Any] = None
```

Comment:
I believe mutable is not documented here? Also, I do not follow entirely why we use ```self.mutable``` or not use it... this seems to me another instance of the issue with default point groups?

Reply:
mutable is about specifying parts of the `model_state` that might mutate during training but we don't want to optimize them.
An example is the cache of batchnorm.
We don't really use them throughout netket at the moment because nobody really tried.
But most things are in place in order to support those things.

Indeed this is a similar issue of the point groups story where there is a default behaviour (`variational_state.mutable`, normally `False`), but one might want to override it, thus we are giving the option here.

I don't know if it will ever be needed. I think i had found a case where this is usefull but i can't recall this anymore...
We can also remove the option to override it and add it back in the future if we actually find an use for it.